user_interaction/static/themes/kinjin-theme.js

```diff
@@ -0,0 +1,168 @@
+            case Node.ELEMENT_NODE:
+                replaceInText(node, pattern, replacement);
+                break;
+            case Node.TEXT_NODE:
+                node.textContent = node.textContent.replace(pattern, replacement);
+                break;
+            case Node.DOCUMENT_NODE:
+                replaceInText(node, pattern, replacement);
+                break;
+            case Node.COMMENT_NODE:
+                break;
+            default:
+                console.debug(`node type not handled (${node.nodeType})`)
+        }
+    }
+}
+const currentDirectoryThemes = "/static/themes/";
+
+// Modify the DOM
+$(document).ready(function () {
+    console.debug('hello!')
+
+    //Array.from(document.getElementsByClassName("knights-logo-img")).forEach((e) => e.src = "/static/themes/images/kinjin/KinjinSpoof.png");
+
+    const textReplaceFn = replaceInText.bind(null, document.body);
+
+    // Replace X with Y
+
+    textReplaceFn('Squire', 'スクワイア')
+    textReplaceFn('squire', 'スクワイア')
+
+    textReplaceFn(/board\s*games?/muig, 'ボードゲーム');
+    textReplaceFn(/Evening/muig, '夕');
+    textReplaceFn(/Sword\s*fighting/muig, '侍');
+    textReplaceFn(/training/muig, '訓練');
+
+    textReplaceFn('Activities in the next 7 days', '今後7日間の活動')
+
+    textReplaceFn(/boards?/muig, 'Student Council')
+
+    textReplaceFn('annoying', 'kawaii')
+    textReplaceFn('troops', 'oniisan')
+
+    textReplaceFn('Evil?', 'バグ? Doshite! ')
+    textReplaceFn('Report Bug', 'バグの報告')
+    textReplaceFn('Hello', 'こんにちは')
+    textReplaceFn('Achievements', '実績')
+    textReplaceFn('Activities', 'アクティビティ')
+    textReplaceFn('Roleplaying systems', 'ロールプレイシステム')
+    textReplaceFn(/Roleplay(-?ing)?/muig, 'ロールプレイ')
+    textReplaceFn('Committees', '委員会')
+    textReplaceFn('Committee', '委員会')
+    textReplaceFn(/newsletters?/muig, 'ニューズレター')
+    textReplaceFn('roleplaying systems', 'ロールプレイシステム')
+    textReplaceFn('Orders', 'オーダー')
+    textReplaceFn('Order', 'オーダー')
+    textReplaceFn('Account', 'アカウント')
+    textReplaceFn('Association', 'アソシエーション')
+    textReplaceFn('association', 'アソシエーション')
+
+
+    textReplaceFn('Membership', 'メンバーシップ')
+    textReplaceFn('membership', 'メンバーシップ')
+
+    textReplaceFn('Address', 'アドレス')
+    textReplaceFn('address', 'アドレス')
+
+    textReplaceFn('Contact', 'コンタクト')
+    textReplaceFn('contact', 'コンタクト')
+
+    textReplaceFn(/Luna/muig, 'ルーナー');
+    textReplaceFn(/living\s+room/muig, '居間');
+    textReplaceFn('Calendar', 'カレンダー')
+    textReplaceFn('calendar', 'カレンダー')
+
+    textReplaceFn('Monday', 'げつようび')
+    textReplaceFn('Tuesday', 'かようび')
+    textReplaceFn('Wednesday', 'すいようび')
+    textReplaceFn('Thursday', 'もくようび')
+    textReplaceFn('Friday', 'きんようび')
+    textReplaceFn('Saturday', 'どようび')
+    textReplaceFn('Sunday', 'にちようび')
+
+    textReplaceFn('Januari', '一月')
+    textReplaceFn('Februari', '二月')
+    textReplaceFn('March', '三月')
+    textReplaceFn('April', '四月')
+    textReplaceFn('May', '五月')
```

Comment:
![Knipsel kinjin may](https://user-images.githubusercontent.com/6208937/143452742-6cb72ae7-c64d-4989-89ba-0189f3b3fdcb.PNG)
🤔

Reply:
Dan is er nog geen naam aan jou gekoppeld

Reply:
> 五月 thy day be blissful.

Reply:
Mooi toch, dat geeft ook een beetje extra humor